client/extensions/woocommerce/app/settings/payments/payment-method-item.js

```diff
@@ -113,12 +150,21 @@ class PaymentMethodItem extends Component {
 		);
 	}
 
+	renderEnabledField = ( isEnabled ) => {
+		return (
+			<PaymentMethodEditFormToggle
```

Comment:
IDK, I'd be inclined to nuke PaymentMethodEditFormToggle and just use the CompactFormToggle here and elsewhere. It doesn't seem PMEFT adds anything to CFT.